.circleci/config.yml

```diff
@@ -411,23 +411,44 @@ jobs:
       - run: mkdir ~/.ssh/ && echo -e "Host github.com\n\tStrictHostKeyChecking no\n" > ~/.ssh/config
       - run: cd packages/web && ./circle_deploy.sh
 
-  test-npm-package-install:
+  test-typescript-npm-package-install:
     working_directory: ~/app
     docker:
       - image: celohq/node8:gcloud
     steps:
       - run:
           name: Installing npm package - @celo/typescript
           command: yarn add @celo/typescript
+
+  test-utils-npm-package-install:
```

Comment:
why several jobs vs several steps?

steps go faster

Reply:
Yes, steps are faster but since this is a cron job and not an every-PR job, speed is not a big concern. One advantage which different jobs have is that instead of `test-npm-package-install` failing. Now `test-<pkgname>-npm-package-install` would fail, so, it is obvious to someone looking at "red" Circle CI as to what job is failing.